src/worker/worker.ml

```diff
@@ -14,7 +14,7 @@ let sync_get url =
         Js.Opt.case
           (File.CoerceTo.arrayBuffer x##.response)
           (fun () ->
-            Firebug.console##log (Js.string "Failed to receive file");
+            Js_of_ocaml.Console.console##log (Js.string "Failed to receive file");
```

Comment:
If compatibility with older js_of_ocaml versions is desired, we could also drop this log...